### PR TITLE
Modified Box Layout GUI Program App

### PR DESCRIPTION
**Pull request by Riston-gitting-gud**
Hi! I may make some mistakes, so please feel free to comment on my code or give any suggestions for improvement, thanks!
@a-rosa
@JevanChandra
@AdriantoHartono
@MikeLeongZhiFai

### CODE REVIEW

File: prac_08/KivyDemos-master/box_layout.kv
Comment (JevanChandra) [inline prac_08/KivyDemos-master/box_layout.kv:1]: Great Layout

File: prac_08/KivyDemos-master/box_layout_demo.py
Comment (JevanChandra) [inline prac_08/KivyDemos-master/box_layout_demo.py:1]: Nice code

File: prac_08/KivyDemos-master/squaring.kv
Comment (JevanChandra) [inline prac_08/KivyDemos-master/squaring.kv:1]: Good Job

File: prac_08/KivyDemos-master/squaring.py
Comment (JevanChandra) [inline prac_08/KivyDemos-master/squaring.py:1]: It works 👍

File: prac_08/convert_miles_km.kv
Comment (JevanChandra) [inline prac_08/convert_miles_km.kv:1]: Very clean Layout

File: prac_08/convert_miles_km.py
Comment (JevanChandra) [inline prac_08/convert_miles_km.py:1]: Nice Code

File: prac_08/dynamic_labels.py
Comment (JevanChandra) [inline prac_08/dynamic_labels.py:1]: Good work

File: prac_08/KivyDemos-master/box_layout_demo.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        self.root.ids.output_label.text = f"Hello {self.root.ids.input_name.text}"
+
+    def press_clear(self):
+        self.root.ids.output_label.text = ""
+        self.root.ids.input_name.text = ""
+
+
+BoxLayoutDemo().run()
+
+
```
Comment (a-rosa) [inline prac_08/KivyDemos-master/box_layout_demo.py:27]: Missing docstring

File: prac_08/KivyDemos-master/squaring.kv [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            # the following line specifies the function in the app class to call when the button is pressed
+            on_press: app.handle_calculate(input_number.text)
+        Label:
+            id: output_label
+            font_size: 48
+            color: (1, 0, 1, 1)
```
Comment (a-rosa) [inline prac_08/KivyDemos-master/squaring.kv:20]: I think this will give out a purple instead of a pink color

File: prac_08/convert_miles_km.kv [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            Button
+                text:'Up'
+                on_press:app.handle_increment(1)
+            Button
+                text:'Down'
+                on_press:app.handle_increment(-1)
+#    BoxLayout:
+#        Button:
+#            text: 'Convert miles to km'
+#            on_press:app.handle_conversion()
```
Comment (a-rosa) [inline prac_08/convert_miles_km.kv]: I think be button section should not be a comment

File: prac_08/convert_miles_km.kv [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    BoxLayout:
+        Label:
+            id: output_km
+            font_size: 48
+            text:app.km
+            color:(1,1,0,1)
```
Comment (a-rosa) [inline prac_08/convert_miles_km.kv:26]: You dont need the boxlayout on top of the layout, just label

File: prac_08/KivyDemos-master/box_layout.kv
Comment (Riston-gitting-gud) [reply prac_08/KivyDemos-master/box_layout.kv:1]: Thank you!

File: prac_08/KivyDemos-master/box_layout_demo.py
Comment (Riston-gitting-gud) [reply prac_08/KivyDemos-master/box_layout_demo.py:1]: Thanks!

File: prac_08/KivyDemos-master/squaring.kv
Comment (Riston-gitting-gud) [reply prac_08/KivyDemos-master/squaring.kv:1]: Thank you!

File: prac_08/KivyDemos-master/squaring.py
Comment (Riston-gitting-gud) [reply prac_08/KivyDemos-master/squaring.py:1]: Good to know. thanks!

File: prac_08/convert_miles_km.kv
Comment (Riston-gitting-gud) [reply prac_08/convert_miles_km.kv:1]: Thank you for the feedback 👍

File: prac_08/convert_miles_km.py
Comment (Riston-gitting-gud) [reply prac_08/convert_miles_km.py:1]: Thank you 👍

File: prac_08/dynamic_labels.py
Comment (Riston-gitting-gud) [reply prac_08/dynamic_labels.py:1]: Thanks 👍

File: prac_08/KivyDemos-master/box_layout_demo.py (hunk 1 above)
Comment (Riston-gitting-gud) [reply prac_08/KivyDemos-master/box_layout_demo.py:27]: Point well taken, I will add docstrings for each function. Thank you for the review!

File: prac_08/KivyDemos-master/squaring.kv (hunk 2 above)
Comment (Riston-gitting-gud) [reply prac_08/KivyDemos-master/squaring.kv:20]: It is a pink colour to me though, but thank you for the feedback 👍

File: prac_08/KivyDemos-master/box_layout.kv
Comment (AdriantoHartono) [reply prac_08/KivyDemos-master/box_layout.kv:1]: Great code

File: prac_08/KivyDemos-master/box_layout_demo.py
Comment (AdriantoHartono) [reply prac_08/KivyDemos-master/box_layout_demo.py:1]: Great code bro👌

**Pull request merged**